Cabal/src/Distribution/Simple/Glob.hs

```diff
@@ -23,6 +23,7 @@ module Distribution.Simple.Glob (
         fileGlobMatches,
         parseFileGlob,
         explainGlobSyntaxError,
+        isRecursiveInRoot,
```

Comment:
Why are we exporting this? Do we use it somewhere else?

Reply:
Yes. I call it in `Cabal/src/Distribution/PackageDescription/Check.hs`.